README.md

```diff
@@ -1,9 +1,47 @@
 # model_seg_rat_axon-myelin_bf
+---
+## Model overview
+
+[//]: # (missing preview image; see https://github.com/axondeepseg/default-BF-model/issues/3)
+
 Default Bright-Field (BF) optical microscopy model that works at a resolution of 0.1 micrometer per pixel.
 
-# Steps to train this model
-1. Get `ivadomed` version: [[83f11d4]](https://github.com/ivadomed/ivadomed/pull/980/commits/83f11d4433f8a8679d11bb7667a4e1d9e02c56b1)
-2. Get the data: `data_axondeepseg_bf_training` (duke/projects/axondeepseg/20211105_bf_datasets/data_axondeepseg_bf_training)
-3. Copy the "model_seg_rat_axon-myelin_bf_training.json" and "split_dataset.joblib" files, and update the following fields: `fname_split`, `path_output`, `path_data` and `gpu_ids`.
-4. Run ivadomed: `ivadomed -c path/to/the/config/file`
-5. The trained model file will be saved under the `path_output` directory.
+## Segment (ADS)
+To segment an image using this model, use the following command in an `axondeepseg` virtual environment:
+```
+axondeepseg -t BF -i <IMG_PATH> -s <PIXEL_SIZE>
+```
+The `-m` option can be omitted in this case because this is a default built-in model.
+
+## Train and test (ivadomed)
+This model was trained and tested with [ivadomed](https://ivadomed.org). We recommend you install ivadomed in a virtual environment to reproduce the original training steps. The specific revision hash of the version used for training is documented in the *version_info.log* file.
+
+### Clone this repository
+You will need the *model_seg_rat_axon-myelin_bf.json* configuration file located in this repo.
+```
+git clone https://github.com/axondeepseg/default-BF-model
+```
+
+### Get the data
+The dataset used to train this model is hosted on git-annex at `data.neuro.polymtl.ca:datasets/data_axondeepseg_bf_training`. The specific dataset revision hash used for training is documented in the *version_info.log* file.
```

Comment:
The dataset was not yet annexed when I did the training of this model so the version_info.log file just says "Dataset is not Annexed."
The dataset version I used is the same as the current one (now annexed), wo we could add the commit of the current version here.